druid-win-shell/src/window.rs

```diff
@@ -1,127 +1,22 @@
-// Copyright 2018 The xi-editor Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-//! Creation and management of windows.
```

Comment:
Retaining a slash-slash-bang would be useful, especially as it's not going to be clear to observers what's the difference between "window" and "windows".